core/kazoo_auth/src/kz_auth_client.erl

```diff
@@ -304,3 +315,56 @@ add_claims(Claims) ->
 
 add_claim(<<"iat">>) -> {iat, kz_time:current_unix_tstamp()-500};
 add_claim(<<"exp">>) -> {exp, kz_time:current_unix_tstamp()+(2 * ?MILLISECONDS_IN_SECOND)}.
+
+-spec request(kz_term:ne_binary(), map()) -> kz_http:ret().
+request(URL, Token) ->
+    request('get', URL, <<>>, Token).
+
+-spec request(atom(), kz_term:ne_binary(), binary(), map()) -> {ok|error, any(), kz_term:proplist()}.
```

Comment:
Return type is wrong. Should be `kz_http:req()` I think.

Reply:
fixing